src/EditorFeatures/CSharpTest/Diagnostics/Nullable/DeclareAsNullableCodeFixTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.CodeFixes.DeclareAsNullable;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics.DeclareAsNullable
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsDeclareAsNullable)]
+    public partial class DeclareAsNullableCodeFixTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    {
+        internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
+            => (null, new DeclareAsNullableCodeFixProvider());
+
+        static readonly TestParameters s_nullableFeature = new TestParameters(parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8));
+
+        [Fact]
+        public async Task FixReturnType()
+        {
+            await TestInRegularAndScript1Async(
+@"class Program
+{
+    static string M()
+    {
+        return [|null|];
+    }
+}",
+@"class Program
+{
+    static string? M()
+    {
+        return null;
+    }
+}", parameters: s_nullableFeature);
+        }
+
+        [Fact]
+        public async Task FixReturnType_ArrowBody()
+        {
+            await TestInRegularAndScript1Async(
+@"class Program
+{
+    static string M() => [|null|];
+}",
+@"class Program
+{
+    static string? M() => null;
+}", parameters: s_nullableFeature);
+        }
+
+        [Fact]
+        public async Task NoFixAlreadyNullableReturnType()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"class Program
+{
+    static string? M()
+    {
+        return [|null|];
+    }
+}", parameters: s_nullableFeature);
+        }
+
+        [Fact]
+        public async Task FixLocalDeclaration()
+        {
+            await TestInRegularAndScript1Async(
+@"class Program
+{
+    static void M()
+    {
+        string x = [|null|];
+    }
+}",
+@"class Program
+{
+    static void M()
+    {
+        string? x = null;
+    }
+}", parameters: s_nullableFeature);
+        }
+
+        [Fact]
+        public async Task NoFixMultiDeclaration()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"class Program
+{
+    static void M()
+    {
+        string x = [|null|], y = null;
+    }
+}", parameters: s_nullableFeature);
+        }
+
+        [Fact]
+        [WorkItem(26628, "https://github.com/dotnet/roslyn/issues/26628")]
+        public async Task FixPropertyDeclaration()
```

Comment:
I think a better approach might be to change the test to the actual desired behavior and add Skip with issue link. I think WorkItem is used for bugs that have been fixed (not for things that still need attention), isn't that the case?

Reply:
I'll change, but `WorkItem` is fine. What matters is there is a tracking issue to drive fixing the problem :-)